.circleci/config.yml

```diff
@@ -4,9 +4,57 @@ version: 2.1
 orbs:
   architect: giantswarm/architect@2.7.0
 
+
+jobs:
+  test-policies:
+    machine: true
+    environment:
+      KIND_VERSION: v0.10.0
+      KUBERNETES_VERSION: v1.19.7
+    steps:
+      - checkout
+      - run:
+          name: Download kind
+          command: |
+            curl -sLo kind https://github.com/kubernetes-sigs/kind/releases/download/$KIND_VERSION/kind-linux-amd64
+      - run:
+          name: Download kubectl
+          command: |
+            curl -sLO https://storage.googleapis.com/kubernetes-release/release/$KUBERNETES_VERSION/bin/linux/amd64/kubectl && chmod +x kind kubectl
+      - run:
+          name: Create cluster
+          command: |
+            ./kind create cluster --image quay.io/giantswarm/kind-node:$KUBERNETES_VERSION --name kyverno-cluster  2> /dev/null
+      - run:
+          name: Wait for the cluster node to be ready
+          command: |
+            ./kubectl wait nodes/kyverno-cluster-control-plane --for=condition=ready --timeout=5m > /dev/null
+            ./kind get kubeconfig --name kyverno-cluster > ./kube.config
+      - run:
+          name: Install Kyverno
+          command: |
+            ./kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml > /dev/null
+      - run:
+          name: Install Giantswarm CRDs
+          command: |
+            ./kubectl create -f https://raw.githubusercontent.com/giantswarm/apiextensions/b9a6bed05850045530eaf8ba7fd01941de6c8525/helm/crds-common/templates/giantswarm.yaml > /dev/null
+      - run:
+          name: Install CAPI Cluster CRD
+          command: |
+            ./kubectl create -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/master/config/crd/bases/cluster.x-k8s.io_clusters.yaml > /dev/null
+            ./kubectl create -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/master/config/crd/bases/cluster.x-k8s.io_machinedeployments.yaml > /dev/null
+      - architect/run-tests-with-abs:
+          chart_dir: "./helm/policies-common"
+          app-build-suite_version: "v0.2.2"
+          app-build-suite_container_tag: "0.2.2"
+          additional_app-build-suite_flags: "--external-cluster-version $KUBERNETES_VERSION"
```

Comment:
`abs` needs the kubernetes version for reasons beyond my comprehension.

Reply:
Its given as an argument to pytest https://github.com/giantswarm/app-build-suite/blob/master/app_build_suite/build_steps/pytest.py#L135

Seems this information is available as `chart_extra_info` fixture then.

Reply:
I'd have thought it could be read from `.abs/main.yaml` instead

Reply:
Flags take precedence over the configuration file. I think this is here because the kubernetes version depend on the ci steps before.